README.md

```diff
@@ -1616,6 +1616,23 @@ Example how to range over 100 objects
 }
 ```
 
+## `int_range [from] [to]`
+
+Returns a `int64` slice whose values range from `from` (inclusive) to `to` (exclusive).
+
+Example how to range from 30 to 50
+
+```django
+{
+    "body":  [
+        {{ range $idx, $v := int_range 30 50 }}
+        ...
+        {{ end }}
+    ]
+}
+```
+In this case `$idx` will iterate from 0 to 19 inclusive, while `$v` will do from 30 to 49 inclusive
+
```

Comment:
We already have the template func "N". Check if you can use that. Should be enough.